update/genesis/export.go

```diff
@@ -317,6 +350,45 @@ func (se *stateExport) exportTx(key string, tx data.TransactionHandler) error {
 	return nil
 }
 
+func (se *stateExport) exportNodesSetupJson(validators map[uint32][]*state.ValidatorInfo) error {
+	acceptedListsForExport := []core.PeerType{core.EligibleList, core.WaitingList, core.JailedList}
```

Comment:
nodes from the core.NewList have already staked and should be added to the waiting Lists in the next epoch. I think these should be also exported.

Reply:
@sasurobert can you please provide an input?

Reply:
ok! no need for new.